spec/src/main/doc/core/injectionandresolution.asciidoc

```diff
@@ -281,27 +279,26 @@ A bean must declare _all_ of the qualifiers that are specified at the injection
 
 [[name_resolution]]
 
-=== EL name resolution
+=== Name resolution
 
-The process of matching a bean to a name used in EL is called _name resolution_. Since there is no typing information available in EL, the container may consider only the bean name. Name resolution usually occurs at runtime, during EL expression evaluation.
+The process of matching a bean to a name is called _name resolution_. Since there is no typing information available during name resolution, the container may consider only the bean name. Name resolution usually occurs at runtime.
 
-An EL name resolves to a bean if:
+A name resolves to a bean if:
 
-* the bean has the given bean name, and
-* the bean is available for injection in the war containing the JSP or JSF page with the EL expression.
+* the bean has the given bean name.
```

Comment:
We should not eliminate the accessibility requirement. Now that we moved from EL name resolution to a more general "name resolution" the accessibility requirement should not only stay but should be made more general, for example:
"the bean is available for injection in the given module"